### PR TITLE
Move libraries to the advanced share menu

### PR DESCRIPTION
**Pull request by jmkulwik**
# Description
This moves the "Share as Library" button into the advanced share menu.

advancedShareDialog2

## Links

- [jira](https://codedotorg.atlassian.net/browse/STAR-873)

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/code-studio/components/AdvancedShareOptions.jsx [hunk 1]
```diff
@@ -362,6 +368,23 @@ class AdvancedShareOptions extends React.Component {
     );
   }
 
+  renderLibraryTab = () => {
+    return (
+      <div>
+        <p style={style.p}>
+          Share the functions in your project with a friend or classmate.
```
Comment (epeach) [inline apps/src/code-studio/components/AdvancedShareOptions.jsx]: i18n?

File: apps/src/code-studio/components/AdvancedShareOptions.jsx [hunk 2]
```diff
@@ -404,12 +433,22 @@ class AdvancedShareOptions extends React.Component {
         ShareOptions.EMBED,
         this.props.i18n.t('project.embed')
       );
+      if (
+        experiments.isEnabled(experiments.STUDENT_LIBRARIES) ||
+        librariesEnabled
```
Comment (epeach) [inline apps/src/code-studio/components/AdvancedShareOptions.jsx:436]: Just to be sure, `librariesEnabled` will be false for the general public at this point, right?

File: apps/src/code-studio/components/AdvancedShareOptions.jsx (hunk 1 above)
Comment (jmkulwik) [reply apps/src/code-studio/components/AdvancedShareOptions.jsx]: Oh woops. Thanks for catching that.

File: apps/src/code-studio/components/AdvancedShareOptions.jsx (hunk 2 above)
Comment (jmkulwik) [reply apps/src/code-studio/components/AdvancedShareOptions.jsx:436]: Yep

**Review by epeach — APPROVED** [review]
Overall, looks good - one 18n reminder and one question to make sure this doesn't go public yet. :)

**Pull request merged**